spring-content-rest/src/test/java/internal/org/springframework/content/rest/links/ContentLinksResourceProcessorIT.java

```diff
@@ -153,6 +155,28 @@ public class ContentLinksResourceProcessorIT {
 					assertThat(resource.getLinks("rendition"), hasItems(hasProperty("href", is("http://localhost/contentApi/testEntity5s/999/rendition"))));
 				});
 			});
+
+			Context("given an entity with an embedded object containing @ContentId properties", () -> {
+				BeforeEach(() -> {
+					PersistentEntity<?, ?> persistentEntity = repositories.getPersistentEntity(TestEntity2.class);
+
+					TestEntity2 obj = new TestEntity2();
+					obj.setId(999L);
+					UUID contentId = UUID.randomUUID();
+					TestEntityChild child = new TestEntityChild();
+					child.setContentId(contentId);
+					obj.setChild(child);
+
+					PersistentEntityResource.Builder build = PersistentEntityResource.build(obj, persistentEntity);
+					resource = build.build();
+				});
+
+				It("should add content property links", () -> {
+					assertThat(resource.getLinks("child"), hasItems(hasProperty("href", is("http://localhost/contentApi/files/999/child"))));
+					assertThat(resource.getLinks("child/content"), hasItems(hasProperty("href", is("http://localhost/contentApi/files/999/child/content"))));
```

Comment:
I am not completely sure about this link name, any different suggestions or conventions (`child.content`?) to name links to nested content properties would be welcome.

Reply:
I am not aware of a standard for alps links in this regard.  What I am doing here is a little bit unique I believe.  So, I *think* it's up to us?  

`child/content` perhaps reflects how the content property as it is addressed when making the request?  i.e. `http://localhost:8080/contacts/12345/child/content`

`child.content` reflects how the content is fetched internally using BeanWrapper and property editors.

Assuming these links are fetched on the consumer side and '/' will not cause any issues I think I prefer the former?

Reply:
If you want to access it from a javascript frontend, both `.` and `/` are equally annoying to deal with (You can't do `response._links.child.href`, but you will have to use `response._links["child/content"].href`) But that's just a minor concern.

So I think I'll keep the slashes in the link name, as that leaves us with less conversion work.